nowplaying/cli.py

```diff
@@ -197,10 +200,6 @@ def get_nonklangbecken_track_handler(self):
             track.observer.IcecastTrackObserver(ICECAST_BASE_URL_10)
         )
 
-        handler.register_observer(
```

Comment:
Are we sure, that no one consumes the RSS output anymore?

Reply:
According to the server logs and to what I remember we stopped using the RSS output hith the move to the current wordpress based website.

Reply:
Ok, fine by me then.